src/main/java/Chunky/Storage/Storage.java

```diff
@@ -0,0 +1,123 @@
+package Chunky.Storage;
+
+import Chunky.Task.Deadlines;
+import Chunky.Task.Events;
+import Chunky.Task.Task;
+import Chunky.Task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Stores and loads the file containing the list of tasks
+ */
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Load file from filePath
+     * @throws FileNotFoundException if filePath leads to a file that doesnt exist
+     * @return main.java.Chunky.Chunky.main.java.Chunky.Task.Task List
+     */
+    public ArrayList<Task> load() throws IOException{
+        ArrayList<Task> lst = new ArrayList<>();
```

Comment:
Instead of lst, use taskList or tasksList for clearer intent about what the list contains.

```suggestion
        ArrayList<Task> taskList = new ArrayList<>();
```